js/app.js

```diff
@@ -0,0 +1,48 @@
+$(function(){
+
+  //listen for a click even on submit and do the function handleSubmit
+  $("#submitBtn").click(handleSubmit);
+
+  //city equals field input, pull the input from field and store it as var city
+  function handleSubmit(event) {
```

Comment:
@sofiaayala this function is not necessary. You have the same logic in the function `changeBackground` (lines 23 -24).